From: Ubuntu Core Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 14 Feb 2019 18:59:14 -0800
Subject: lib-argparse

# DP: argparse.py: Make the gettext import conditional
# DP: argparse.py: Make the gettext import conditional
---
 Lib/argparse.py | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/Lib/argparse.py b/Lib/argparse.py
index fd61bc7..9e7542b 100644
--- a/Lib/argparse.py
+++ b/Lib/argparse.py
@@ -89,7 +89,16 @@ import re as _re
 import shutil as _shutil
 import sys as _sys
 
-from gettext import gettext as _, ngettext
+try:
+    from gettext import gettext as _, ngettext
+except ImportError:
+    def _(message):
+        return message
+    def ngettext(singular,plural,n):
+        if n == 1:
+            return singular
+        else:
+            return plural
 
 SUPPRESS = '==SUPPRESS=='
 
